src/pages/about/governance.md

```diff
@@ -1,44 +1,51 @@
+- Working Group Members
+- Tauri Board & Board Directors
+- Domains & Domain Leads
+- Teams
 
-The current working groups are:
+![Tauri governance diagram](./governance-diagram.svg)
 
-- WG Governance & Guidance
-- WG Tech
-- WG Education
-- WG Media
-- WG Security
-- WG Devops
 
-With the exception of the security working group, which is by invite only and convenes privately, all other working groups are public and open to any and all participants.
+### Working Group Members
 
-Visit the [Governance and Guideance repository](https://github.com/tauri-apps/governance-and-guidance) to get more information.
+All of the individuals that make up the Tauri Working Group.
 
-## Code of Conduct
+### Tauri Board & Board Directors
 
-Everyone participating in the Tauri community is expected to follow the [Code of Conduct](https://github.com/tauri-apps/governance-and-guidance/blob/main/CODE_OF_CONDUCT.md).
+The collective Board is the central decision making body for the Tauri Programme. Although the Tauri Board is ultimately responsible for decisions, they may delegate to Domain Leads as they see fit.
+
+Individual Board Directors may not be directly involved in all areas and aspects of but they are responsible for performing Board duties to the best of their abilities.
+
+### Domains & Domain Leads
+
+Domains are organizational units that represent an area of interest within Tauri.
+
+Domain Leads are trusted contributors within the Tauri community with expertise in the Domain they are leading. They are responsible for setting direction, overseeing and supporting the activities within that Domain.
 
-## Social Contract
+The current Domains and Domain leads are outlined in the [Governance and Guidance repo on GitHub](https://github.com/tauri-apps/governance-and-guidance).
 
-We have a [Social Contract](https://github.com/tauri-apps/governance-and-guidance/blob/main/SOCIAL_CONTRACT.md) that informs our decision making and organization.
+### Teams
 
-## Licensing
+Teams are small groups of contributors that support or maintain specific areas of the Tauri Programme. They are a means for Tauri to execute its longer-term tasks and goals, especially when ad-hoc contributions cannot achieve the same results.
 
-We, the contributors to Tauri Apps, use the MIT and Apache licenses for all code content. Images and bodies of text, unless otherwise noted are CC-BY-ND-NC.
 
-## Trademark
+## Get Involved
 
-It is a permissible use of the name "Tauri App" or the Tauri logo to show that a project uses Tauri. "Tauri Studio" is reserved for use by the organization.
+Anyone is welcome to request joining the Tauri Working Group. You can reach out on the [Tauri Discord](https://discord.com/invite/tauri) or send an email to [board@tauri.app](mailto:board@tauri.app).
```

Comment:
Oh please no :joy:

I feel like:
1. WG access shouldn't be decided *just* by the Board.
2. The WG itself is really fuzzy, so the CTA should concentrate on Teams first.
3. Current WG members are invited / endorsed by someone else, it represents a trust-level that I think we shouldn't change here.

Reply:
Phrasing might have been a bit off (it was end of the week for me 😅) but I meant it to be:

1) If you don't use Discord you can email board as an alternative means
2) You mean to illustrate the interest-specific places for someone to get involved in? I'd start off with domains since those are more mature in practice than teams atm
3) Hmm, I see your point. But I'm also wondering how we have someone express interest in joining without them having "buddied up" with someone if that makes sense

Lmk what you think about those and can work to tweak it a bit.

Reply:
Ah gotcha, let me revise that

Reply:
Just removed that paragraph/sentence